savoirfairelinux/sous-chef
# Create a field to add a emergency contact relationship

## Fixes #490 

### Changes proposed in this pull request:

Add a new field on the form emergency contact to define an emergency contact relationship

### Status


### How to verify this change

Create /update a client. On emergency client form, you should see a new field to enter a relationship

### Deployment notes and migration


### New translatable strings
